mlir/lib/ExecutionEngine/CMakeLists.txt

```diff
@@ -168,8 +168,8 @@ if(LLVM_ENABLE_PIC)
   set_property(TARGET mlir_float16_utils PROPERTY CXX_STANDARD 17)
   target_compile_definitions(mlir_float16_utils PRIVATE mlir_float16_utils_EXPORTS)
 
-  if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
-    # TODO: This support library is only used on Linux builds until we figure
+  if(CMAKE_SYSTEM_NAME STREQUAL "Linux" OR CMAKE_SYSTEM_NAME STREQUAL "Darwin")
```

Comment:
nit: I would put all the Darwin-related changes into a separate PR.

Reply:
done